taichi/gui/gui.h

```diff
@@ -859,6 +852,10 @@ class GUI : public GUIBase {
     }
   }
 
+  Vector2 pos_coord_uniform(Vector2i pos) {
```

Comment:
```suggestion
  Vector2 canvas_untransform(Vector2i pos) {
```
We may also add `canvas_transform` for future use?

Reply:
Thanks for your patient advices ! 
And do you mean that just rename to ```canvas_untransform``` here , or some other difference between these two functions ?